package.json

```diff
@@ -36,7 +36,9 @@
 		"@sapphire/result": "^1.1.1",
 		"@sapphire/stopwatch": "^1.4.1",
 		"@sapphire/utilities": "^3.7.0",
+		"@types/object-hash": "^2.2.1",
```

Comment:
types in regular deps?

Reply:
yeah otherwise people need to install them themselves or use skipLibCheck and seeing as we're a type-first framework that would be bad DX

Reply:
ah, thanks